### DISCUSSION TITLE
Issues with Lively not running video: No associated plugin?

### DISCUSSION THREAD

**Discussion by earthshoes** [rocksdanister/lively#1894] [Q&A]
I've been using Lively as a way to run an animated desktop (video of a national park I'm fond of) for at least a year--probably longer. I've had no issues at all until recently. It is telling me that there's no associated plugin with the file (MP4), which makes no sense at all. I've tried several different things, including restarting, uninstalling and reinstalling Lively, associating the file with different video programs, etc. And it's still kicking out the same error. 

I've got to be honest with you--this video is the only reason I have Lively. Otherwise, I'm not big on animated desktops. I thought it was possible that AVG removed something during a clean up, but if so reinstalling everything should work. 

Please advise.

**Comment by rocksdanister** [comment]
https://github.com/rocksdanister/lively/wiki/Common-Problems#having-a-problem-how-do-i-get-help--how-to-retrieve-log-files
